pyrokinetics/pyro.py

```diff
@@ -31,8 +32,12 @@ def __init__(
         self.supported_local_geometries = ["Miller", None]
 
         self.gk_file = gk_file
-        self.gk_type = gk_type
-        self.gk_code = self.gk_type
+        if gk_type is not None and gk_code is None:
+            raise DeprecationWarning(
+                "gk_type is no longer used, please use gk_code instead"
+            )
```

Comment:
Should probably also keep `self.gk_code = gk_type` for now and remove in a future version?

Reply:
How do I "raise" the warning without stopping the program. Can't seem to figure that out

Reply:
The `warnings` module is what you want:

https://docs.python.org/3/library/warnings.html#warnings.warn

This users filter the warning out